pkg/project/v2/project_loader_test.go

```diff
@@ -569,3 +570,21 @@ func getFullProjectLoaderContext(apis []string, projects []string, environments
 		ParametersSerde: config.DefaultParameterParsers,
 	}
 }
+
+func TestLoadProject(t *testing.T) {
+	t.Run("loading mixed projects", func(t *testing.T) {
```

Comment:
nit: if this test fails a month or two from now, I'll have no clue how "loading mixed projects" should work.
IMO would be nicer to name this more directly like the test projects "loading project with configs and account resources works" and "mixing configs and account resources in same file fails" - so the name alone tells me how things should have worked